html/about.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>Tanisha Hampden</title>
+  <link rel="stylesheet" href="../styles/main.css">
+  <link rel="stylesheet" href="../styles/about.css">
+  <script src="https://use.fontawesome.com/9ff7ff7a76.js"></script>
+  <style>@import url('https://fonts.googleapis.com/css?family=Ubuntu+Mono');</style>
+</head>
+<body>
+  <!--  -->
+  <!-- header -->
+  <!--  -->
+  <header>
+    <nav>
+      <h1><a class="home-link" href="index.html">{T|H}</a></h1>
+      <ul>
+        <li><a href="index.html">home</a></li>
+        <li><a href="portfolio.html">portfolio</a></li>
+        <li><a href="code-journal.html">blog</a></li>
+        <li><a href="about.html">about</a></li>
+      </ul>
+    </nav>
+  </header>
+
+  <!--  -->
+  <!-- main -->
+  <!--  -->
+  <main>
+    <h3>a litte about me...</h3>
+    <p>I am a student softwore developer at <a href="https://adadevelopersacademy.org/">Ada Developer's Academy</a>. My personal mission is to get more Black and Latino Women into tech. I am interested in storytelling, travel, and self-improvement. Before begin Ada, I had just concluded a year-long journey through Europe and Vietnam, respectively. While traveling I spent my time thinking, writing and learning to code. <a href="portfolio.html">Have a look</a> at what I am currently working on.</p>
+
+    <p>I have an educational background in Sociology and I am currently interested in using code and data to better frame and answer big questions (i.e. income inequality). I also have interests in writing software that automates everyday tasks and collects and analyzes personal data, product design, and open source software.</p>
+
+  </main>
+
+  <!--  -->
+  <!-- footer -->
+  <!--  -->
+  <footer class="site-footer">
+    <div class="footer-wrapper">
+      <div class="copyright">
+        <p>This is a blog and portfolio by Tanisha Hampden. Built with love and Atom.</p>
+
+        <p> Copyright &copy; Tanisha Hampden 2017. All Right Reserved.</p>
+
+      </div>
+
+      <div class="contact">
+        <a href="https://github.com/tanham">
+          <i class="fa fa-github fa-2x fa-border " aria-hidden="true "></i>
```

Comment:
Why are these `<i>` (italic) tags? Seems like `<img>` might be what you want here?

Reply:
These tags are specific to the font-awesome cdn that I added. The font-awesome docs require you to use them for the social media icons to show up. I think the `i` in this context might stand for icon.